src/ray/rpc/server_call.h

```diff
@@ -298,6 +302,10 @@ class ServerCallImpl : public ServerCall {
   /// Human-readable name for this RPC call.
   std::string call_name_;
 
+  /// ID of the cluster to check incoming RPC calls against.
+  /// Check skipped if empty.
```

Comment:
Should we check cluster for all the rpc calls happening in the cluster except the RegisterClient one?

Reply:
Yes, that's in the followup PR! This one was getting too big, lol